src/types/index.tsx

```diff
@@ -26,7 +26,8 @@ export interface ResourcesLinkProps {
   type: "link" | "download";
   icon?: string;
   details: {
-    description: string;
+    title: string;
+    description?: string;
```

Comment:
date?

Reply:
I put description in case in the future it's used for something else 😅